src/games/strategy/triplea/delegate/MoveValidator.java

```diff
@@ -1709,13 +1734,13 @@ public static Route getBestRoute(final Territory start, final Territory end, fin
     List<Match<Territory>> tests;
     if (isNeutralsImpassable) {
       tests = new ArrayList<Match<Territory>>(Arrays.asList(
-          // best if no enemy and no neutral
+      // best if no enemy and no neutral
```

Comment:
Looks like a tab was removed - good. Makes me wonder what we were doing with a tab in this file. Though, it does look like the indention is now off.